### ISSUE TITLE
Power flow loss issues

### ISSUE THREAD

**Issue by viktor-ktorvi** [viktor-ktorvi/mlpf#5]
There are a few things I'd like to sort out.

* The test fails on a few topologies due to size mismatches.
* I'd also like to test it out on more samples like on my generated ppcs as well as on the nets of powerdata-gen.
* I'm also interested in implementing the loss as a multiplication of spatse matrices and using timeit to compare the two approaches cause the matrix one is easier to use and could be useful in the future

**Comment by viktor-ktorvi** [comment]
Here's one implementation in the matrix form. It could be refined:

```python
def power_flow_loss(ppc):
    ppc = pypower.api.ext2int(ppc)
    baseMVA, bus, gen, branch = ppc["baseMVA"], ppc["bus"], ppc["gen"], ppc["branch"]
    ref, pv, pq = pypower.api.bustypes(bus, gen)
    Sbus = pypower.api.makeSbus(baseMVA, bus, gen)
    Ybus, _, _ = pypower.api.makeYbus(baseMVA, bus, branch)
    V = bus[:, 7] * np.exp(1j * np.pi / 180 * bus[:, 8])
    mis = V * np.conj(Ybus * V) - Sbus
    F = np.r_[mis[pv].real, mis[pq].real, mis[pq].imag]
    loss = np.sum(np.abs(F))
    return loss
```
